packages/opentelemetry-core/test/resource.test.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import { Resource } from '../src/resources/Resource';
+
+describe('Resource', () => {
+  const resource1 = new Resource({
```

Comment:
Should these constants be capitalized since they are not meant to be mutated?

Reply:
I don't think so. These are local variables that just happen to be constant. I personally use capitalization only when they are actual file or project level constants.